_includes/footer.md

```diff
@@ -4,10 +4,7 @@
 [![Apache Software Foundation][ASF_IMG]{: #asf-logo height="100"}][ASF]
 
 Copyright &copy; 2011-{{ site.time | date: '%Y' }} The Apache Software
-Foundation, Licensed under the [Apache License, Version 2.0][AL2]. Apache
```

Comment:
Did you move this block about trademarks somewhere? It looks like it was deleted -- we definitely do not want to do that.